lib/foreman_discovery/host_converter.rb

```diff
@@ -0,0 +1,26 @@
+class ForemanDiscovery::HostConverter
```

Comment:
I thought we discouraged putting this kind of thing in `lib/` and instead prefer it in `app/services` ? Not too worried either way, but if that's the convention in core, then plugins should probably follow it too.